utils/near-performance-metrics/src/stats_enabled.rs

```diff
@@ -42,7 +42,7 @@ struct Entry {
 }
 
 pub struct ThreadStats {
-    stat: HashMap<(&'static str, u32), Entry>,
+    stat: HashMap<(&'static str, u32, &'static str), Entry>,
```

Comment:
At this point we might want to introduce a struct with name fields for `(&'static str, u32, &'static str)`

Reply:
We could, but we are not using any of the fields by name. We simply print those 3 fields in that order.